dotCMS/src/main/java/com/dotmarketing/db/DbConnectionFactory.java

```diff
@@ -114,6 +115,30 @@ public static Connection getConnection() {
     }
 
     /**
+     * Returns true if the connection is already create and will be reuse in a transaction.
+     * Otherwise will returns false, that means a new connection will be created.
+     * @return boolean
+     */
+    public static boolean connectionExists() {
+
+        boolean isCreated = false;
+        final Map<String, Connection> connectionsMap =
+                (HashMap<String, Connection>) connectionsHolder.get();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "HashMap". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)